src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -215,6 +220,16 @@ public RemoteExecutionService(
     this.captureCorruptedOutputsDir = captureCorruptedOutputsDir;
 
     this.scheduler = Schedulers.from(executor, /*interruptibleWorker=*/ true);
+
+    String regex = remoteOptions.experimentalForceDownloadsRegex;
+    // TODO(bazel-team): Consider adding a warning or more validation if the experimentalForceDownloadsRegex is used
+    // without RemoteOutputsMode.MINIMAL
+    this.shouldForceDownloads = !regex.isEmpty() && remoteOptions.remoteOutputsMode == RemoteOutputsMode.MINIMAL;
```

Comment:
I think we can also use this flag for `--remote_download_toplevel`.

Reply:
Done